config/initializers/lograge.rb

```diff
@@ -21,3 +22,13 @@ def fmt.call(data)
     end
   end
 end
+
+ActiveSupport::Notifications.subscribe(/cache_read.*\.active_support/) do |name, start, finish, arg1, arg2|
+  RequestStore[:cache_stats] ||= {}
+  RequestStore[:cache_stats][:read] ||= 0
+  RequestStore[:cache_stats][:read] += 1
+end
+
+ActiveSupport::Notifications.subscribe(/cache_.*\.active_support/) do |name, start, finish, arg1, arg2|
```

Comment:
Is one of these `.subscribe` calls meant to be a `.instrument`? Or is that happening elsewhere?

Reply:
Nope, these are subscribing to built in instrumentation [ActiveSupport](https://edgeguides.rubyonrails.org/active_support_instrumentation.html#active-support) has for caching.